<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C&amp;C Server Commander</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Network Traffic Collector</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="ntc_100">
     <property name="toolTipDuration">
      <number>6</number>
     </property>
     <attribute name="title">
      <string>100</string>
     </attribute>
     <widget class="QLineEdit" name="sip1001">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>22</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start IP :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>62</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>End IP :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip1002">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip1003">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lip1004">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn100_collect">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Collect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn100_send">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn100_watch">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Watch</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn100_copy">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn100_close">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip1004">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cmd100">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn100_cmd">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Send CMD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnSet100ip">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set IP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnOpenTargetList_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>125</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Traffic Collection Target List .txt File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gcp_TargetListPath_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: 9998</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ntc_200">
     <attribute name="title">
      <string>200</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>62</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>End IP :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>22</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start IP :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn200_send">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn200_watch">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Watch</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn200_collect">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Collect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn200_copy">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn200_close">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip2003">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip2004">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lip2004">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip2002">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sip2001">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cmd200">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtn200_cmd">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Send CMD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnSet200ip">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set IP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gcp_TargetListPath_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnOpenTargetList_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>125</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Traffic Collection Target List .txt File</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ntc_gcp">
     <attribute name="title">
      <string>GCP</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>GCP PC IP List .txt File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnSetGCPip">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set IP (Open)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_send">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_watch">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Watch</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_collect">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Collect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_copy">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_close">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gcp_TargetListPath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>95</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Traffic Collection Target List .txt File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gcp_ipListPath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_OpenTargetList">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnGCP_cmd">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Send CMD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cmdGCP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: 9998</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>541</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Instance Controller</string>
    </property>
    <widget class="QPushButton" name="InCtl_btnConnect">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send CMD</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>501</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="InCtl_ipInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="InCtl_cmd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="InCtl_user">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="InCtl_pw">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Target IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
